python/pytorchserver/README.md

```diff
@@ -0,0 +1,146 @@
+# PyTorch Server
+
+[PyTorch](https://PyTorch.org) server is an implementation of KFServing for serving PyTorch models, and provides a PyTorch model implementation for prediction, pre and post processing.
+
+To start the server locally for development needs, run the following command under this folder in your github repository.
+
+```
+pip install -e .
```

Comment:
make dev-install?
Can you update the other docs you copied this from, as well?

Reply:
I like the pip syntax on top for python server. Allows me to switch to pip3 et all.

Reply:
Shouldn't you be aliasing this :?